request-handler/src/main/java/au/org/aodn/aws/wps/lambda/WpsLambdaRequestHandler.java

```diff
@@ -35,90 +28,30 @@ public AwsApiResponse handleRequest(AwsApiRequest request, Context context) {
         WpsRequestHandler handler = new WpsRequestHandler();
         AwsApiResponse response = null;
 
-        //  Read environment variables
-        //  Identify the location of the configuration file to use
-        String s3RegionName = getEnvironmentVariable(REGION_NAME_ENV_VARIABLE_NAME);
-        String bucketName = getEnvironmentVariable(S3_BUCKET_ENV_VARIABLE_NAME);
-        String configFilename = getEnvironmentVariable(CONFIG_FILENAME_ENV_VARIABLE_NAME);
-
-
-        AmazonS3Client s3Client = new AmazonS3Client();
-        Region region = Region.getRegion(Regions.fromName(s3RegionName));
-        s3Client.setRegion(region);
-
-        String envName = getEnvironmentName(context);
-
-        //  Append an environment name to the front of the config filename
-        if(envName != null)
-        {
-            configFilename  = envName + "/" + configFilename;
-        }
-
-        LOGGER.log("S3 Config location: " + bucketName + "/" + configFilename);
 
-        //  Note:  the Lambda function needs access to read from S3 bucket location
-        //  that contains the configuration file.
-        S3Object configFile = s3Client.getObject(bucketName, configFilename);
-        S3ObjectInputStream contentStream = configFile.getObjectContent();
-        Properties config = new Properties();
         try
         {
-            //  Load configuration from S3 file
-            config.load(contentStream);
+            String envName = getEnvironmentName(context);
+            Properties config = WpsConfig.getConfigProperties(envName);
+
+            LOGGER.log("Loaded configuration from S3.");
 
             //  Execute the request
             response = handler.handleRequest(request, config);
 
         }
-        catch(IOException ioex)
+        catch(Exception ex)
         {
             //  Bad stuff happened
-            LOGGER.log("Exception running WPS Lambda function: " + ioex.getMessage());
+            LOGGER.log("Exception running WPS Lambda function: " + ex.getMessage());
```

Comment:
Do we need to setup an error reponse here?

Reply:
Yep - will add TODO

Reply:
Added TODO